// [python quickstart](https://grpc.io/docs/quickstart/python.html#run-a-grpc-application)
// python -m grpc_tools.protoc --python_out=. --grpc_python_out=. -I. xxxxxxx.proto


syntax = "proto3";

service Translate {
    rpc infer(TranslateRequest) returns (TranslateReply) {}
}

message TranslateRequest {
    bytes input_bytes = 1;
}

message TranslateReply {
    bytes result_bytes = 1;
}


